src/content/en/updates/2019/01/shape-detection.md

```diff
@@ -93,10 +96,13 @@ text. The following bullet list contains examples of use cases for all three fea
 | ------------------------------------------ | ---------------------------- |
 | 1. Create explainer                        | [Complete][explainer]        |
 | 2. Create initial draft of specification   | [In Progress][spec]          |
-| 3. Gather feedback & iterate on design     | [In progress](#feedback)     |
-| **4. Origin trial**                        | [**In progress**](ot)        |
+| 3. **Gather feedback & iterate on design** | [**In progress**](#feedback) |
```

Comment:
Maybe we should duplicate this table for each of the features?

Reply:
In the mid-term we should, especially since text detection got outsourced in an independent spec. For now, I think it's fine, as *gather feedback & iterate on design* is true for both.